docs/pages/deploy-a-cluster/helm-deployments/aws.mdx

```diff
@@ -306,7 +287,15 @@ depending on whether you are running PostgreSQL or MongoDB, to your values file:
   ```
 
 </TabItem>
-<TabItem label="Your own TLS credentials">
+<TabItem label="Your own TLS">
```

Comment:
"Your own TLS" sounds a bit awkward to me. Can you own transport layer security?

Reply:
That's fair, I think most users reading this guide know they don't own the transport layer 😄 

```suggestion
<TabItem label="Your own TLS credentials">
```